src/null-safety/index.md

```diff
@@ -5,37 +5,36 @@ description: Information about Dart's null safety feature
 
 The Dart language comes with sound null safety. 
 
-In Dart 3, you always get null safety. In Dart 2.x,
-it must be enabled [with a pubspec setting](#enable-null-safety).
-
 Null safety prevents errors that result from unintentional access
 of variables set to `null`.
+
 For example, if a method expects an integer but receives `null`,
-your app causes a runtime error. This type of error, a null dereference error,
-can be difficult to debug.
-
-With sound null safety variables are 'non-nullable' by default:
-They can be assigned only values of the declared type
-(e.g. `int i=42`), and never be assigned `null`.
-You can specify that a type of a variable is nullable
-(e.g. `int? i`),
-and only then can they contain either a `null` *or*
+your app returns a runtime error.
+This type of error, a null dereference error, can be difficult to debug.
+
+With sound null safety, variables default to 'non-nullable'.
+You can assigned values of the declared type only, like `int i=42`.
+You can never assign the value to `null` to default variable types.
+To specify that a variable type can be nullable, you add a `?` after
+the type label: `int? i`.
+These specific types can contain either a `null` *or*
 a value of the defined type.
 
 Sound null safety changes potential **runtime errors**
-into **edit-time** analysis errors, by flagging when
-any non-nullable variable hasn't been initialized with a 
-non-null value or is being assigned a `null`.
-This allows you to fix these errors before deploying your app.
+into **edit-time** analysis errors.
+Null safety does flagging that a non-null variable has either:
+
+* Not been initialized with a non-null value
+* Been assigned a `null` value.
 
+This check allows you to fix these errors _before_ deploying your app.
 
 ## Introduction through examples
 
-With null safety,
-all of the variables in the following code are non-nullable:
+With null safety, all variables in the following code can't be `null`:
 
 ```dart
-// In null-safe Dart, none of these can ever be null.
+// In Dart 3, none of these can ever be null.
```

Comment:
This is a historical document and it also applies to null-safe code before Dart 3.

```suggestion
// With null safety, none of these can ever be null.
```